run.sh

```diff
@@ -14,5 +14,5 @@ migrate)
     bundle exec rake db:schema:load
     ;;
 esac
-bundle exec import:all assets:precompile
+RAILS_ENV=production bundle exec import:all assets:precompile
```

Comment:
https://github.com/ministryofjustice/prison-visits-2/blob/master/config/database.yml#L26
https://github.com/ministryofjustice/prison-visits-2/blob/master/config/database.yml#L81

Rails environments were created for development, testing and production. If we will enforce RAILS_ENV production during every container run ( even during jenkins build / tests ) there might be a problem with at least connectivity. It counts for email servers details as well.